ops/model.py

```diff
@@ -691,6 +691,32 @@ def set_ports(self, *ports: Union[int, 'Port']) -> None:
         for protocol, port in desired - existing:
             self._backend.open_port(protocol, port)
 
+    def reboot(self, now: bool = False) -> None:
+        """Reboot the host machine, after stopping all containers hosted on the machine.
```

Comment:
What does "after stopping all containers" mean in this context, particularly given that this doesn't work on K8s?

Reply:
> What does "after stopping all containers" mean in this context, particularly given that this doesn't work on K8s?

It's stolen from the [juju docs](https://juju.is/docs/sdk/hook-tool#heading--juju-reboot). I'm not 100% sure on the meaning. I think [this is the relevant Juju code](https://github.com/juju/juju/blob/f3aed6169084eb326a397da071399581947b89a8/cmd/jujud/reboot/reboot.go#L58), but it's not clear to me what `containers` means in that context.

Reply:
Hmm, I have no idea what that comment means either (it's 9 years old, well before we started supporting K8s, so containers must have meant something different). I'd drop it until/unless we understand it. Feel free to ask on the Juju channel (ping `wallyworld`) if you want, or we can just drop it.

Reply:
[Asked](https://chat.charmhub.io/charmhub/pl/qshza6ucetntjma18nixqbpayc).